web/components/Members.tsx

```diff
@@ -4,7 +4,7 @@ import PropTypes, {InferProps} from 'prop-types'
 
 function Members({teamData} : InferProps<typeof Members.propTypes>) {
   const {members} = teamData
-  const memberItems = members.map((member: any) => <Member key={member.name} image={`/../public/assets/RoleLogos/${member.position.toLowerCase()}.png`} name={member.name} position={member.position} />)
```

Comment:
public is implied in deployment, so even though it would self correct in dev, in deployment it would cause errors because the Image API was fetching in the wrong place.